ethereum/contract_addresses.go

```diff
@@ -82,6 +86,8 @@ var ChainIDToContractAddresses = map[int]ContractAddresses{
 		DevUtils:            common.HexToAddress("0x04674b391d3a58f4350e5306d5251a74e91f65da"),
 		WETH9:               common.HexToAddress("0xc778417e063141139fce010982780140aa0cd5ab"),
 		ZRXToken:            common.HexToAddress("0xff67881f8d12f372d91baae9752eb3631ff0ed00"),
+		ChaiBridge:          common.HexToAddress("0x0000000000000000000000000000000000000000"),
+		ChaiToken:           common.HexToAddress("0x0000000000000000000000000000000000000000"),
```

Comment:
@abandeali1 do you know what the contract addresses are for these other testnets, including our Ganache snapshot?

Reply:
I'm not aware of Chai being deployed on any of the testnets, so we also did not deploy the `ChaiBridge` contract. Is that a blocker at all?

Reply:
We could pretty easily deploy Chai and ChaiBridge on Kovan, since MCD is on there. The others might be a pain in the ass though...

Reply:
It's probably not a blocker, I'll just reject Chai Bridge orders on other testnets for now so that we don't accept Chai Bridge orders where the bridge address is set to the null address.

Reply:
@mzhu25 could we at least deploy on our Ganache snapshot so we can write a test for this?

Reply:
Turns out this is difficult. Punting on this for now.